### PR TITLE
IE8 8.0.6001.18702 workaround

### PR DESCRIPTION
**Pull request by kroko**
adding undefined to fight this issue https://github.com/eligrey/classList.js/issues/36
modernie IE8-MSWXP machine has IE8 8.0.6001.18702 and is affected

### CODE REVIEW

File: classList.js [hunk 1]
```diff
@@ -173,7 +173,9 @@ if (objCtr.defineProperty) {
 	try {
 		objCtr.defineProperty(elemCtrProto, classListProp, classListPropDesc);
 	} catch (ex) { // IE 8 doesn't support enumerable:true
-		if (ex.number === -0x7FF5EC54) {
+		// adding undefined to fight this issue https://github.com/eligrey/classList.js/issues/36
+		// modernie IE8-MSW7 machine has IE8 8.0.6001.18702 and is affected
```
Comment (nbouvrette) [inline classList.js]: // modernie -> // modern ?

Comment (kroko) [inline classList.js]: modernie = htttp://www.modern.ie/ = webpage, where MS hosts free virtual machines, so one can test webcode on OSses from MSWXP up to MSW10, IE6 up to IE11 & Edge. the page now gets redirected to https://developer.microsoft.com/en-us/microsoft-edge/ (this is more than one year old pull request :)) and the actual page now for virtual machines is https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/ where the lowest now is WinXP w/ IE8.

Comment (nbouvrette) [inline classList.js]: My bad, I thought it was a typo. Good to know regarding modern.ie.

Comment (englishextra) [inline classList.js]: Yeah. it was about the MS virtual images distributed by modernie (WHICH IS NOW  https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) where devs can download images to test for old ie's

**Pull request merged**